test_runner/src/main/kotlin/ftl/gc/GcStorage.kt

```diff
@@ -93,6 +95,17 @@ object GcStorage {
             logLn("Cannot upload performance metrics ${it.message}")
         }.getOrNull()
 
+    fun uploadMatricesId(args: IArgs, matrixMap: MatrixMap): String {
+        val matrixIdsPath: Path = if (args.useLocalResultDir())
+            Paths.get(args.localResultDir, FtlConstants.matrixIdsFile) else
+            Paths.get(args.localResultDir, matrixMap.runPath, FtlConstants.matrixIdsFile)
```

Comment:
The same logic exists in `updateMatrixFile`, IMO we can have a function for getting matrixIdsPath. For example `fun IArgs.matrixIdsPath(matrixMap: MatrixMap): Path`

Reply:
Sounds good!